SamplesDashboard/SamplesDashboardTests/RepositoriesServiceTests.cs

```diff
@@ -82,7 +134,7 @@ public async Task ShouldGetDependencies()
             var packageManager = "NUGET";
 
             //Act
-            var dependencies = await _sampleService.GetRepository(name);
+            var dependencies = await _repositoriesService.GetRepository(name);
```

Comment:
rename `dependencies` to `repository` as the function returns a repository object